.github/workflows/wsl-helper-build.yaml

```diff
@@ -0,0 +1,45 @@
+# This workflow builds the WSL Helper
+name: 'GitHub Runner: WSL Helper'
```

Comment:
Is this just for linting? Would it make sense to eventually make this one script that covers `src/go/*` instead?

Reply:
Yeah, it would be nice to have the same config file for all the projects but right now I'm not 100% sure what each configuration would look like, some may need windows build some don't, etc.